vagrant/provisioning/roles/arkcase-app/templates/server.yml

```diff
@@ -78,6 +78,7 @@ alfresco_port: 443
 email:
   should-delete-messages: false
   should-mark-messages-as-read: true
+  receiver-channel-enabled: {{receiver-channel-enabled | false}}
```

Comment:
Should it be `{{ receiver-channel-enabled | default(false) }}`?

Reply:
yes, addressed the comment